Sources/AppStoreConnectCLI/Model/BetaTester.swift

```diff
@@ -29,34 +29,20 @@ struct BetaTester: ResultRenderable {
         self.apps = apps
     }
 
-    init(_ betaTester: AppStoreConnect_Swift_SDK.BetaTester, _ includes: [AppStoreConnect_Swift_SDK.BetaTesterRelationship]?) {
-        let attributes = betaTester.attributes
-        let relationships = betaTester.relationships
-
-        let includedApps = includes?.compactMap { relationship -> AppStoreConnect_Swift_SDK.App? in
-            if case let .app(app) = relationship {
-                return app
-            }
-            return nil
-        }
-
-        let includedBetaGroups = includes?.compactMap { relationship -> AppStoreConnect_Swift_SDK.BetaGroup? in
-            if case let .betaGroup(betaGroup) = relationship {
-                return betaGroup
-            }
-            return nil
-        }
+    init(with output: GetBetaTesterOperation.Output) {
```

Comment:
For an init function that directly converts one type to another with can ommit the `with` keyword